apps/coscrad-frontend/src/components/resources/playlists/playlist-detail.thumbnail.presenter.tsx.tsx

```diff
@@ -0,0 +1,7 @@
+import { ICategorizableDetailQueryResult, IPlayListViewModel } from '@coscrad/api-interfaces';
+
+export const PlaylistDetailThumbnailPresenter =
+    // eslint-disable-next-line no-empty-pattern
```

Comment:
Instead of `{}:ICategorizable...` try `_:ICategorizable....`, i.e. use `_` as your placeholder. Then you won't need the eslint disable line.